src/utils/mail.py

```diff
@@ -19,6 +17,7 @@
 DEFAULT_BODY = "Enjoy!"
 DEFAULT_SUBJECT = "New free packt ebook"
 
+
 class MailBook:

```

Comment:
PEP8 requires two empty lines before class declaration - would you like to lint that code using ex. [pylama](https://github.com/klen/pylama)?